app/assets/stylesheets/common/base/alert.scss

```diff
@@ -40,3 +40,27 @@
 a.alert.clickable {
   display: flex;
 }
+
+.alert-bootstrap-mode {
+  display: flex;
+  flex-direction: row;
+  padding: 10px;
+
+  .col-text {
+    width: 85%;
```

Comment:
Similar comment to the other PR, this probably doesn't look great on mobile. And if we aren't showing this message on mobile, then we should move the styles to a `desktop/` stylesheet.

Reply:
The banner was indeed looking bad on mobile. I've now added separate styling for mobile and desktop. Here's how the mobile banner looks now:

<img width="392" alt="Screenshot 2022-08-09 at 9 42 01 PM" src="https://user-images.githubusercontent.com/5732281/183704489-9569645b-fb09-4029-a368-52f1d10d1464.png">